pandas-stubs/core/reshape/concat.pyi

```diff
@@ -9,13 +9,12 @@ from typing import (
 )
 
 from pandas._typing import (
-    Axis as Axis,
-    DataFrame as DataFrame,
-    Series as Series,
```

Comment:
I don't think we want to encourage people doing something like this `from pandas.core.reshape.concat import DataFrame`

Reply:
The `DataFrame as DataFrame` and `Series as Series` appears in LOTS of the files, so at some point those should get cleaned up.